src/packages/sdk/src/IntrospectionEngine.ts

```diff
@@ -202,10 +244,14 @@ export class IntrospectionEngine {
         try {
           const binaryPath = await resolveBinary('introspection-engine')
           debugRpc('starting introspection engine with binary: ' + binaryPath)
-          this.child = spawn(binaryPath, {
-            stdio: ['pipe', 'pipe', 'pipe'],
+
+          const flags = [...this.flags]
```

Comment:
Here we could save the `[...` as we don't mutate it anymore
```suggestion
          const { flags }  = this
```